### PR TITLE
Fix undo bugs

### PR DESCRIPTION
**Pull request by turtle96**
should fix bug issues #120 and #109

### CODE REVIEW

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/documentation 
@CS2103AUG2016-W15-C3/code-quality 
@CS2103AUG2016-W15-C3/testing

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Comment by Mamtha3005** [comment]
The 2 issues (#120 , #109) are unsolved. However, all Junit tests pass.

File: src/main/java/seedu/taskell/logic/commands/UndoCommand.java [hunk 1]
```diff
@@ -54,7 +65,35 @@ public UndoCommand(int index) {
...
+                commandHistoryList.remove(commandHistory);
+            }
+        }
+        
+    }
+
+    private boolean isCommandTypeAddOrEdit(CommandHistory commandHistory) {
+        return (commandHistory.getCommandType().contains(AddCommand.COMMAND_WORD) 
+                || commandHistory.getCommandType().contains("edit")) 
+                && !commandHistory.isRedoTrue();
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/logic/commands/UndoCommand.java:90]: Magic string. Please use constant in EditCommand class
image

File: src/main/java/seedu/taskell/logic/commands/UndoCommand.java [hunk 2]
```diff
@@ -54,7 +65,35 @@ public UndoCommand(int index) {
...
+    }
     
+    private boolean isUndoEditCommand(CommandHistory commandHistory) {
+        return commandHistory.isRedoTrue() 
+                && commandHistory.getCommandType().contains("edit");
+    }
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/logic/commands/UndoCommand.java:96]: Magic string. Please use constant in EditCommand class or create a constant

**Comment by marcusngwenjian** [comment]
Code quality reviewed. Please make the necessary changes

**Pull request merged**